packages/vitest/src/runtime/execute.ts

```diff
@@ -64,7 +63,8 @@ export interface ContextExecutorOptions {
 }
 
 export async function startVitestExecutor(options: ContextExecutorOptions) {
-  const state = () => getWorkerState() || options.state
+  // @ts-expect-error injected untyped global
+  const state = () => globalThis.__vitest_worker__ || options.state
```

Comment:
What's the reason for these changes?

Reply:
getState now throws an error. here it's possible that it's not initialized (in vm for example)